hack/gen-resources/generators/application_generator.go

```diff
@@ -49,8 +49,8 @@ func (ag *ApplicationGenerator) buildSource(opts *util.GenerateOpts, repositorie
 }
 
 func (pg *ApplicationGenerator) buildRandomDestination(opts *util.GenerateOpts, clusters []v1alpha1.Cluster) (*v1alpha1.ApplicationDestination, error) {
-	rand.Seed(time.Now().Unix())
-	clusterNumber := rand.Int() % len(clusters)
+	seed := rand.New(rand.NewSource(time.Now().Unix()))
```

Comment:
2 things about `rand.NewSource()` usage:
- will this be backward compatible with the previous implementation?
- just like `rand.Seed`, the docs warn about `rand.New` not being concurrency safe. Can you please double check?

Reply:
1. Yep, we just need randomness, don't really care how.
2. This code is not run concurrently.

This code is a utility for load-testing Argo CD. It's not run as part of Argo CD itself.